doc/api.md

```diff
@@ -1916,6 +1919,45 @@ the referenced resource(s). In some cases, this permission cannot
 be determined and the value will be `undefined`.
 
 **Kind**: instance property of <code>[Reference](#ERMrest.Reference)</code>  
+<a name="ERMrest.Reference+display"></a>
+
+#### reference.display : <code>Object</code>
+An object which contains row display type for this reference. 
+ Will be populated on basis of  "table-display" annotation. 
+
+ The object has following properties
+ {
+   
+   rowOrder: [{ column: "NAME", descending: true/false }] || undefined,
+   
+   type: "markdown",  // Possible values are table/markdown/module (Default is "table")
+
```

Comment:
can you guys communicate? I asked @RFSH to review this pull request also; that should help.